commisery/cli/__init__.py

```diff
@@ -246,10 +249,10 @@ def check(ctx, target):
             )
             > 1
         ):
-            log.debug(f"Handling as range: {target_str}")
+            log.debug(f"Handling as range: {target_str}")  # pylint: disable= W1203
             result = check_commit_rev_range(target, config=config)
         else:
-            log.debug(f"Handling as commitish: {target_str}")
+            log.debug(f"Handling as commitish: {target_str}")  # pylint: disable= W1203
```

Comment:
What's the warning that you're disabling here?

Reply:
String formatting for loggers should use `%s`, will change